tests/ase/test_utils.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING
+
+import pytest
+from ase.build import bulk
+from ase.calculators.lj import LennardJones
+from ase.optimize import BFGS
+from ase.spacegroup.symmetrize import check_symmetry
+from numpy.testing import assert_allclose
+
+if TYPE_CHECKING:
+    from pymatgen.core import Structure
+
+
+from atomate2.ase.utils import AseRelaxer, FrechetCellFilter, TrajectoryObserver
+
+
+def test_safe_import():
+    assert FrechetCellFilter is None or FrechetCellFilter.__module__ == "ase.filters"
```

Comment:
do we still need this test? given that we pin `"ase>=3.23.0"` in pyproject, `FrechetCellFilter` will be importable always

Reply:
Don't think so - removed this and the logic around safely trying to import `FrechetCellFilter`